sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 1.10.0-beta.2 (Unreleased)
 
 ### Features Added
-- Add `BrowserCustomizedOptions` to `InteractiveBrowserCredential` to enable web view customization for interactive authentication.
+- Added `BrowserCustomizations` property to `InteractiveBrowserCredential` to enable web view customization for interactive authentication.
```

Comment:
This is plural, but the actual API is singular `BrowserCustomization`. Personally, I like the plural `BrowserCustomizations` better, but the singular was what was suggested in the API review.

Reply:
Sticking with the singular, so I'll change it here.